runtime/Dart/lib/src/atn/src/atn.dart

```diff
@@ -92,9 +92,9 @@ class ATN {
     return s.decision;
   }
 
-  DecisionState getDecisionState(int decision) {
-    if (decisionToState.isNotEmpty) {
-      return decisionToState[decision];
+  DecisionState? getDecisionState(int? decision) {
```

Comment:
we only have this parameter as nullable, because there is a place where we call:

```dart
predicateDFAState(D, atn.getDecisionState(dfa.decision));
```

And DFA decision is nullable.